inttest/common/bootloosesuite.go

```diff
@@ -905,6 +906,15 @@ func (s *BootlooseSuite) ExtensionsClient(node string, k0sKubeconfigArgs ...stri
 	return extclient.NewForConfig(cfg)
 }
 
+// KubeClient return kube client by loading the admin access config from given node
+func (s *BootlooseSuite) HelmClient(node string, k0sKubeconfigArgs ...string) (*helmclient.HelmV1beta1Client, error) {
```

Comment:
I'd rather not add anything more to BootlooseSuite. That struct got a little bit out of hand over time and is already hard to improve without a massive refactoring. For historical reasons, it mixes up several aspects that would rather be handled in separate structs. The most problematic part is the implicit usage of the test context over the struct's methods, instead of having the context as first parameter.

Reply:
Fair enough, added it as a method to the OpenEBSSuite